README.md

```diff
@@ -1,8 +1,8 @@
 # Syllabus
-https://docs.google.com/presentation/d/1xaw0AXhimgnBBTctfTFHHIHddcI1RBUM0ITjexDiRRo
+https://docs.google.com/presentation/d/1w8HRmAK2HB5PuwOB0HdFeGbvntsvM3-rvHdNOPZ3lBs
 
 # Before you begin
-Install all dependencies and link executables to child projects with `yarn` or `npm i` in the `exercises` folder.
+Install all dependencies and link executables to child projects with `npm i` in the `exercises` folder.
```

Comment:
Since both me and @branoholy failed to run npm i on the first try, I'd suggest to move the package.json to the root directory. What do you think?

Reply:
This will result in the same kind or problem as we encountered while updating `concurrently` package: the [link-parent-bin](https://github.com/nicojs/node-link-parent-bin) package will not update the binaries of parent dependencies as we would like. In this case it wont even try to add links in exercise directories (00-09) during `npm i`, as they are not located in neither `.` nor `packages` directory related to the root of the project. When overriding the child expected directory with `-c exercises` argument the binaries are not updated for those of us who cloned the old version (with package.json in exercises directory) and already installed the packages from there:
```
Linking dependencies ["concurrently","link-parent-bin","nodemon","prettier"] under children ["00-init","01-react-stateful","02-react-stateless","03-redux","04-react-redux","05-reselect","06-redux-saga","07-normalizr","08-router5","09-forms","node_modules"]
09:51:51 (7409) INFO link Different link at 'exercises/01-react-stateful/node_modules/.bin/prettier' already exists. Leaving it alone, the package is probably already installed in the child package.
...
```

Reply:
@novellizator @branoholy @dinassh  There is a lot of magic going on with the solution above. Considering that this is a training project, we should keep the non-react stuff down to the minimum.

I think that the idea of having a single package.json file is great, but linking dependencies with some magical symlinking not so much :-) For the sake of simplicity I suggest this:
1. Replace CRA with Parcel bundler
2. Have one package.json file in the root directory
3. Have multiple start commands specified, i.e. `start:0` would do `parcel src/00-init/index.html` etc.

With the above, we should have one set of dependencies but still be able to invoke the app from multiple entry points, which is what we want to do unless I am really mistaken.

AFAIK Parcel should do out of the box everything that CRA does, maybe except ESlint stuff.

What do you think?

Reply:
@karelklima FYI during today's discussion it was proposed to move all exercise directories (00-09) up to the root

1. I don't have much experience with either of them, so I don't have the preference at this point.
2. I propose to address this issue in another pull request.

Reply:
Good idea worth considering! We didn't plan to address it in this PR per se. Let me create a card in the project and we'll address it down the road